src/main/resources/com/google/api/codegen/csharp/gapic_snippets.snip

```diff
@@ -67,7 +67,7 @@
 @private flattenedMethodSync(snippetMethod)
     public void {@snippetMethod.snippetMethodName}()
     {
-        // Snippet: {@snippetMethod.method.name}({@snippetArgTypes(snippetMethod.method.methodParams)},CallSettings)
+        // Snippet: {@snippetMethod.method.name}({@snippetArgTypes(snippetMethod.method.methodParams)}{@if(snippetMethod.method.methodParams, ",", "")}CallSettings)
```

Comment:
Probably the logic like this should be in the view object itself or in the `@snippetArgTypes()` "snip" method (so you can avoid repeating the `@if` statement everywhere).

Reply:
So `@snippetArgTypes()` are reused in another overload which has an additional argument `CancellationToken` (see [here](https://github.com/googleapis/gapic-generator/blob/cd919793e439d2d21505900aa08ef9f99519d8fa/src/main/resources/com/google/api/codegen/csharp/gapic_snippets.snip#L85)). Moving `CallingSettings` into the view object will cause huge refactoring to C# templates. Given the state of the C# micro generator it is probably not worth it...